### ISSUE TITLE
Website feedback when proof is completed?

### ISSUE THREAD

**Issue by ospfranco** [tlsnotary/explorer#26]
Hey, trying to verify a proof of mine but the website appears to do nothing?

![OSP 000196](https://github.com/user-attachments/assets/c2128ba8-3707-4eea-8694-33bec994a227)

Not sure if the proof failed? or passed? or is it computing? Am I doing anything wrong?

Any feedback is appreciatted!

**Comment by heeckhau** [comment]
Did you try and click the "Next" button?

**Comment by ospfranco** [comment]
yes, I clicked the next button, that's why the console shows the message. I'm not using a local version, I'm using the deployed web version.

**Comment by ospfranco** [comment]
No confidential info, it's just a dummy test account

[calm_proof.json](https://github.com/user-attachments/files/16600067/calm_proof.json)

public key:

-----BEGIN PUBLIC KEY-----
MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEX6DaRpTdcNDvOC/RONHOaPUgO/It
IsQT4USlhnXubc29Z6RIYGxMX11eYJwaiJKzbdiWbt2FEEetj1aCArNpJw==
-----END PUBLIC KEY-----

**Comment by heeckhau** [comment]
Looks like you have the wrong notary key. We should show a better message for this.

@Codetrauma can you take a look at this? https://tlsnotary.github.io/proof_viz/ renders this as:
<img width="1248" alt="image" src="https://github.com/user-attachments/assets/12e31ec2-9b3f-456b-a04e-89f26df05285">
